src/command_line_parser.cc

```diff
@@ -1060,6 +1072,10 @@ TritonParser::Parse(int argc, char** argv)
         break;
       case OPTION_HTTP_ADDRESS:
         lparams.http_address_ = optarg;
+        break;
+      case OPTION_HTTP_HEADER_FORWARD_PREFIX:
+        lparams.http_forward_header_pattern_ = optarg;
```

Comment:
Can we create lhttp_options struct like for grpc_option below in L1207?

Reply:
I think it can be but it requires changes in vertex and others. I have created a ticket for this refactor: https://jirasw.nvidia.com/browse/DLIS-4758